src/cs-graphics/HDRBuffer.cpp

```diff
@@ -105,10 +105,10 @@ void HDRBuffer::bind() {
     // NOLINTNEXTLINE(cppcoreguidelines-owning-memory)
     hdrBuffer.mLuminanceMipMap = new LuminanceMipMap(mMultiSamples, size[0], size[1]);
 
-    // Create glow mipmaps.
-    delete hdrBuffer.mGlowMipMap; // NOLINT(cppcoreguidelines-owning-memory)
+    // Create glare mipmaps.
+    delete hdrBuffer.mGlareMipMap; // NOLINT(cppcoreguidelines-owning-memory)
     // NOLINTNEXTLINE(cppcoreguidelines-owning-memory)
-    hdrBuffer.mGlowMipMap = new GlowMipMap(mMultiSamples, size[0], size[1]);
+    hdrBuffer.mGlareMipMap = new GlareMipMap(mMultiSamples, size[0], size[1]);
```

Comment:
This manual memory management seems dangerous. Not just here, but throughout the whole file.

I also can't find a **final** `delete` for this and some other manually managed pointers. Do they leak memory?

Reply:
You're right. This is an relic and should be fixed.